src/Models/user.model.ts

```diff
@@ -1,11 +1,17 @@
 import {Model} from 'objection';
+import { Country } from '../Types/country';
 
-export default class User extends Model {
-    static tableName = 'users';
-    id!: number;
-    user_name!: string;
-    email!: string;
-    password! : string;
-    phone_number!: string;
-    country!: string;
+
+export interface UserType {
+  id: number;
+  user_name: string;
+  email: string;
+  password: string;
+  phone_number: string;
+  country: Country;
+}
+export class User extends Model{
+  static get tableName() {
+    return 'users';
```

Comment:
Declare strings like this as a constant string and import them

Reply:
In a different file?

Reply:
Yes